features/order/managing_orders/seeing_order_shipment_state_as_shipped_if_there_is_not_shipments_to_deliver.feature

```diff
@@ -0,0 +1,28 @@
+@managing_orders
+Feature: Seeing shipping states of an order as shipped if there is no shipments to deliver
+    In order to get to know the state of shipping for order without shipments
```

Comment:
`In order to have a coherent behaviour of order shipment states`?

Reply:
And something similar for the feature above?

Reply:
```
In order to have coherent shipping states of all orders
As an Administrator
I want orders with no shipments to deliver to have shipping state shipped
```